docs/canonicalk8s/snap/reference/crypto.md

```diff
@@ -0,0 +1,87 @@
+# Cryptography
```

Comment:
As you have added this as a explanation page, we can remove the reference page. We can create another card for a crypto reference page if you like but for now this can go.